examples/go-migrations/migrations/00002_rename_root.go

```diff
@@ -1,8 +1,7 @@
-package main
+package migrations
```

Comment:
Undo due to recommendation above

Reply:
The problem is that this is compiled into it's own binary so that the migrations are not in your main application at all.

Reply:
So, this is basically an example of how you would set up your migrations. Ideally you would copy this entire folder into your project and replace the migrations with your own. If it was main, your migrations would have to be in the same directory as your executable code.